### ISSUE TITLE
Viewing Accuracy/IoU and images with bounding boxes for TF2.0 object detection 

### ISSUE THREAD

**Issue by lamdawr** [tensorflow/models#8162]
System information

Have I written custom code (as opposed to using example directory): https://github.com/tensorflow/models/tree/master/official/vision/detection changed strategy to "mirrored" to run in GPU/CPU, reduced batch size to 1 to start with minimum and changed location to my custom tfrecord. all other code unchanged. reference for other files : #8119
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): ubuntu 18.04.2 LTS bionic
TensorFlow backend (yes / no): yes
TensorFlow version: 2.1.0
Keras version: 2.3.1
Python version: 3.7.3
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Describe the current behavior: total loss is Nan and Error occurred when finalizing GeneratorDataset iterator

logs: Model Training gives only the following oututs : 'total_loss', 'cls_loss', 'box_loss', 'model_loss', 'l2_regularization_loss'

Describe the expected behavior

Previous object detection API would display IOU, images with bounding box and the label along with the confidence of the value.

Code to reproduce the issue: same code available in https://github.com/tensorflow/models/tree/master/official/vision/detection except for running it locally with strategy as "mirrored" and config file : #8119Describe the expected behavior
Calling model train should start training

Code to reproduce the issue: same code available in https://github.com/tensorflow/models/tree/master/official/vision/detection except for running it locally with strategy as "mirrored" and config file : #8119

**Comment by haimat** [comment]
How can you make this Object Detection API work with TF2, I tought it's for TF1.x only?!

**Comment by laxmareddyp** [comment]
Hi @lamdawr,

Current object detection API is displaying IOU, images with bounding box and the label along with the confidence of the value.Please check and let us know from your end.

Thanks.
